src/pages/documentation/profile/api-reference.md

```diff
@@ -118,12 +130,16 @@ Remember the following information:
 * If the attribute exists, the value will be updated.
 * A null attribute value removes the attribute.
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="5"/>
 
-iOS (ACP 2.x)
+iOS
 
 <Tabs query="platform=ios-acp&api=update-user-attribute"/>
 
+React Native
+
+<Tabs query="platform=react-native&api=update-user-attribute"/>
```

Comment:
@hbhandal80 the react native additions in this commit don't seem to be defined under src/pages/documentation/profile/tabs/ , do you plan on adding them in the next PRs?

Reply:
please refer to the link provided in https://github.com/AdobeDocs/acp-mobile-sdkdocs/pull/8